.github/workflows/pr-branch-build.yml

```diff
@@ -19,3 +19,10 @@ jobs:
           yarn lint
           yarn test
           yarn build
+      - name: Dry Run Publish
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{secrets.LIFEOMIC_NPM_TOKEN}}
+        run: |
+          yarn build
+          npx semantic-release --dry-run
```

Comment:
Using `--dry-run` here will cause it to log out the "expected" version bump. Pretty nifty.